sorc/ncep_post.fd/MDL2P.f

```diff
@@ -1816,6 +1831,101 @@ SUBROUTINE MDL2P(iostatusD3D)
           ENDIF
         ENDIF
 !     
+!***  CHIPSI
+!
+        IF (IGET(1021) > 0 .or. IGET(1022) > 0) THEN
+          IF (LVLS(LP,IGET(1021)) > 0 .or. LVLS(LP,IGET(1022)) > 0) THEN
+            CALL CALCHIPSI(USL,VSL,CHI,PSI)
+!         print *,'me=',me,'EGRID1=',EGRID1(1:10,JSTA)
+!     
+!*** CHI 
+!
+          IF (LVLS(LP,IGET(1021)) > 0) THEN  
+!$omp  parallel do private(i,j)
+             DO J=JSTA,JEND
+               DO I=ISTA,IEND
+                 GRID1(I,J) = CHI(I,J)
+               ENDDO
+             ENDDO
+
+            IF (SMFLAG .or. ioform == 'binarympiio' ) THEN
```

Comment:
@KarinaAsmar-NOAA @JesseMeng-NOAA May I know the reason of applying the smoothc function for velocity potential calculation?

Reply:
@KarinaAsmar-NOAA Please remove all smoothing processing.